tic-tac-toe.js

```diff
@@ -1,7 +1,83 @@
-function TicTacToe() {
+$(document).on('ready', function() {
+  var game = new TicTacToe();
 
-}
+// click handler - keep it separate
+// we could have saved the button clicking function to a variable and then passed that variable to click
+  $(":button").click( function(){
+    if (game.turns % 2 === 0) {
+      $(this).css('background-color', 'red');
+    } else {
+      $(this).css('background-color', 'green');
+    }
+    // this gives me the class that the person has clicked on and is storing the value so I can use it elsewhere
+    var position = $(this).attr("class").split(" ")[1];
+    game.play(position, game.marker());
+  });
+});
 
-TicTacToe.prototype = {
-  
-}
+  // creates a class that initializes constructor for tictactoe class
+  // create a function for each of these functions?
+  function TicTacToe() {
+    this.board = [];
+    this.player = "player1";
+    this.turns = 0;
+  }
+
+  TicTacToe.prototype.play = function(position, marker) {
+  this.turn();
```

Comment:
It seems like you would want to switch the players at the end of a turn instead of at the beginning of the turn, right? Will this then end up with player 2 as the first player?